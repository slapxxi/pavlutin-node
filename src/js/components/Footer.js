import React from 'react';
import ExternalLink from './ExternalLink';
import { Links, Container, Copy } from './Footer.styled';
import { links } from './SocialLinks';
import { currentYear } from '../utils';

function Footer() {
  return (
    <footer>
      <Container>
        <Links />
        <p>
          This site is an {' '}
          <ExternalLink href={links.openSource}>
            open-source
          </ExternalLink>{' '}
          project.
          Coded in {' '}
          <ExternalLink href="https://atom.io">
            Atom
          </ExternalLink>. {' '}
          Built using {' '}
          <ExternalLink href="https://facebook.github.io/react/">
            React
          </ExternalLink>{' '}
          and {' '}
          <ExternalLink href="http://redux.js.org">
            Redux
          </ExternalLink>{' '}
          with {' '}
          <ExternalLink href="https://styled-components.com">
            Styled Components
          </ExternalLink>.
        </p>
        <Copy>
          Slava Pavlutin &copy; 2016-{ currentYear() }
        </Copy>
      </Container>
    </footer>
  );
}

export default Footer;
