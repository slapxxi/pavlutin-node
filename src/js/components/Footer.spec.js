import React from 'react';
import render from 'react-test-renderer';
import Footer from './Footer';

it('renders site footer', () => {
  const tree = render.create(<Footer />);
  expect(tree.toJSON()).toMatchSnapshot();
});
