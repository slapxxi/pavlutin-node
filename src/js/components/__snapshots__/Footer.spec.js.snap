// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders site footer 1`] = `
<footer>
  <div
    className="labIeh"
  >
    <nav
      className="nav euufCD"
    >
      <ul>
        <li
          className="nav__item"
        >
          <a
            href="https://twitter.com/SlavaPavlutin"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              aria-hidden={true}
              className="fa fa-twitter"
            />
             
             
            Twitter
          </a>
        </li>
        <li
          className="nav__item"
        >
          <a
            href="https://github.com/slavapavlutin"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              aria-hidden={true}
              className="fa fa-github"
            />
             
             
            GitHub
          </a>
        </li>
        <li
          className="nav__item"
        >
          <a
            href="https://dribbble.com/slavapavlutin"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              aria-hidden={true}
              className="fa fa-dribbble"
            />
             
             
            Dribbble
          </a>
        </li>
        <li
          className="nav__item"
        >
          <a
            href="mailto://sl.pavlutin@gmail.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              aria-hidden={true}
              className="fa fa-envelope-o"
            />
             
             
            Email
          </a>
        </li>
      </ul>
    </nav>
    <p>
      This site is an 
       
      <a
        href="https://github.com/slavapavlutin/pavlutin-node"
        rel="noopener noreferrer"
        target="_blank"
      >
        open-source
      </a>
       
      project. Coded in 
       
      <a
        href="https://atom.io"
        rel="noopener noreferrer"
        target="_blank"
      >
        Atom
      </a>
      . 
       
      Built using 
       
      <a
        href="https://facebook.github.io/react/"
        rel="noopener noreferrer"
        target="_blank"
      >
        React
      </a>
       
      and 
       
      <a
        href="http://redux.js.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Redux
      </a>
       
      with 
       
      <a
        href="https://styled-components.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        Styled Components
      </a>
      .
    </p>
    <small
      className="dweWkS"
    >
      Slava Pavlutin Â© 2016-
      2017
    </small>
  </div>
</footer>
`;
